src/model/Series.ts

```diff
@@ -519,7 +519,18 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
         if (!selectedMap) {
             return;
         }
+        const selectedMode = this.option.selectedMode;
+
         const data = this.getData(dataType);
+        if (selectedMode === 'series') {
+            data.each(idx => {
```

Comment:
It's simpler and faster to just assign an empty object to `selectedMap` and `_selectedDataIndicesMap` if wan't to unselect all.